src/shared/tpm2-util.c

```diff
@@ -4204,7 +4204,7 @@ int tpm2_tpm2b_public_from_openssl_pkey(const EVP_PKEY *pkey, TPM2B_PUBLIC *ret)
 
         TPMT_PUBLIC public = {
                 .nameAlg = TPM2_ALG_SHA256,
-                .objectAttributes = TPMA_OBJECT_DECRYPT | TPMA_OBJECT_SIGN_ENCRYPT | TPMA_OBJECT_USERWITHAUTH,
+                .objectAttributes = TPMA_OBJECT_DECRYPT | TPMA_OBJECT_SIGN_ENCRYPT | TPMA_OBJECT_USERWITHAUTH | TPMA_OBJECT_NODA,
```

Comment:
uh, unless I am dumb this will break compat with all previous enrollments of pubkey policies. That's just not acceptable. We *must* retain compat here, we cannot just lock everyone out of their systems.

It's fine to set the flag for new enrollments, but we must be able to unlock old enrollments too.

Reply:
Turns out that the lack of this attribute here does not affect the ability to enroll nor unlock while under DA lockout. The key's only used to verify a signature, it's not used for signing/encrypting/decrypting, hence why.

I've removed the attribute.